source/includes/_transactions.md

```diff
@@ -77,18 +82,32 @@ message Signature {
 }
 ```
 
-**Payload** stores everything except signatures: 
+**Payload** stores all transaction fields, except signatures: 
 <ul>
     <li> Time of creation (unix time, in milliseconds) </li> 
-    <li> ID of transaction creator (username@domain) </li>
-    <li> Transaction counter — counts how many times transaction creator send transactions in total. Counter is used to prevent replays, and is formed on client side. </li>
-    <li> Repeated commands which would be described in details later</li> 
+    <li> Account ID of transaction creator (username@domain) </li>
+    <li> Transaction counter. It counts how many times transaction creator send transactions in total. Counter is used to prevent replay attack, and is formed on client side </li>
+    <li> Repeated commands which are described in details in <a href="#commands">commands section</a> </li> 
 </ul>
 
 **Signatures** contain one or many signatures (ed25519 pubkey + signature):
 
 <aside class="notice">
-In preview version, there is no way to get current transaction counter by any means. You have to remember it on a client side and increment it. In future version this will be changed, and client application may retrieve this information.
+In current version, there is no way to get current transaction counter by any means. You have to remember it on a client side and increment it. In future version this will be changed, and client application may retrieve this information.
 </aside>
 
 ## Transaction statuses
+
+Current version of Iroha peer follows client [pull principle](https://en.wikipedia.org/wiki/Pull_technology) for networking. It means that client should be proactive and request the state of transaction from peer. 
+
+This section describes the set of states and mathes them with transaction lifecycle.
+
+![Block](../images/tx_status.png "Block structure")
+
+ * `NOT_RECEIVED`: as client can query the peer, which may not have this transaction, the response indicates about such situation.
```

Comment:
As far as I know, if peer does not know about transaction means there is no such transaction at all.